crates/core/tedge_api/src/entity_store.rs

```diff
@@ -164,12 +164,27 @@ impl EntityStore {
         self.entities.get(entity_topic_id)
     }
 
-    /// Returns information for an entity under a given device/service id .
+    /// Tries to get information about an entity under a given MQTT entity topic identifier.
+    pub fn try_get(&self, entity_topic_id: &EntityTopicId) -> Result<&EntityMetadata, Error> {
```

Comment:
Pedantic comment: Returning `entity_store::Error` here is a bit of an overkill as this API only returns a single type of error. So, it could have been a dedicated error struct.

But, let's be practical and don't go too far with error types. So, let it be.